languages/julia/exercises/concept/functions-introduction/.docs/after.md

```diff
@@ -0,0 +1,42 @@
+### Summary
```

Comment:
The introduction has a nice mention about the two different ways to define functions, but I don't see that described here. As part of the goal of the `after.md` document is to serve as a reference for students to go back to later, I think the after document should at least contain all the information from the `introduction.md` document, possibly with additional resources or a more in-depth explanation.

Reply:
I didn't want to duplicate any information, but I'll copy the missing parts over.